test/fixtures/v4/did-signed-wrapped.json

```diff
@@ -0,0 +1,51 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://schemata.openattestation.com/com/openattestation/4.0/alpha-context.json"
+  ],
+  "type": ["VerifiableCredential", "OpenAttestationCredential"],
+  "issuanceDate": "2021-03-08T12:00:00+08:00",
+  "validFrom": "2021-03-08T12:00:00+08:00",
```

Comment:
do we still need `issuanceDate` since `validFrom` is now [prefered](https://www.w3.org/2018/credentials/#issuanceDate)?

Reply:
Oh yes, I forgot to remove the `issuanceDate` field. Thanks for pointing out!